docs/src/docs/asciidoc/user/examples.adoc

```diff
@@ -44,8 +44,7 @@ cd ehcache3/demos/00-NoCache
 ----
 
 This builds the necessary components, starts a http://eclipse.org/jetty/[Jetty] web service,
-and displays the URL of the web server on the console.  The URL will be something like
-+http://localhost:8080/ehcache-demos/00-NoCache/+.
+and displays the URL of the web server on the console. The URL should be +http://localhost:8080/+.
```

Comment:
Would it make sense to set the port anyway to ensure these instructions continue to match?

Reply:
If you find the field :-) Otherwise, I'm fine with the default

Reply:
Since I'm a nice guy, I found the new field. So it is now explicit again in the `build.gradle`